debian/changelog

```diff
@@ -1,15 +1,15 @@
-kolibri-source (0.12.2-0ubuntu2) trustry; urgency=medium
+kolibri-source (0.12.2-0ubuntu2) trusty; urgency=medium
```

Comment:
omg!

Reply:
sorry I didn't review your PR carefully and missed this typo.

Reply:
hahha